src/components/map/hooks/use-map-setup.tsx

```diff
@@ -157,7 +157,10 @@ export function useMapSetup(
 		});
 		initializedMap.addControl(geoLocateControl, "bottom-left");
 		initializedMap.addControl(
-			new mapboxgl.NavigationControl({ showCompass: false, showZoom: true }),
+			new mapboxgl.NavigationControl({
+				showCompass: false,
+				showZoom: !isMobile,
```

Comment:
Here is an unhandled edge-case: when the screen size becomes bigger/smaller (e.g. on changing orientation), the buttons won't adapt. But I think we can ignore this edge-case so far.

Reply:
Good point. Since the zoom functionality is still possible with touch interaction or scrolling I also think we can ignore it for now.